packages/core/test/StringField_test.js

```diff
@@ -221,7 +236,7 @@ describe("StringField", () => {
       });
 
       sinon.assert.calledWithMatch(onChange.lastCall, {
-        formData: undefined,
+        formData: "",
```

Comment:
These were previously undefined because `options.emptyValue` was undefined. If the user enters the empty string, I believe we should call `onChange` with the empty string.